test/integration/admin_can_modify_only_their_account_data_test.rb

```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.